deps/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-if(TARGET mapbox-base)
```

Comment:
We need this - this is how we avoid multiple target definitions of `mapbox-base` shared between cascading dependencies.

Reply:
It is in a root CMake? Is that ok?
https://github.com/mapbox/mapbox-base/blob/kdn-dev/CMakeLists.txt#L16-L18